protocol/contracts/nexus/ERC721ACustom.sol

```diff
@@ -0,0 +1,1150 @@
+// SPDX-License-Identifier: MIT
+// ERC721A Contracts v4.2.3
+// Creator: Chiru Labs
+
+pragma solidity 0.8.18;
+// Temple (nexus/ERC721ACustom.sol)
+
+import "../interfaces/nexus/IERC721A.sol";
+
+/**
+ * @dev Interface of ERC721 token receiver.
+ */
+interface ERC721A__IERC721Receiver {
+    function onERC721Received(
+        address operator,
+        address from,
+        uint256 tokenId,
+        bytes calldata data
+    ) external returns (bytes4);
+}
+
+/**
+ * @title ERC721A
```

Comment:
How does this differ from `https://github.com/chiru-labs/ERC721A/blob/main/contracts/ERC721A.sol`?

Could you add a comment to this say that it's a direct fork of that (gh perma link) and then what the differences are and why?